notebooks/304-pytorch-post-training-quantization/304-pytorch-post-training-quantization.ipynb

```diff
@@ -0,0 +1,612 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Post-Training Quantization by NNCF\n",
+    "\n",
+    "This notebook is based on 'ImageNet training in PyTorch' example.\n",
+    "\n",
+    "The goal of this notebook is to demonstrate how to use the Neural Network Compression Framework NNCF 8-bit quantization to optimize a PyTorch model for inference with OpenVINO Toolkit. The optimization process contains the following steps:\n",
```

Comment:
Let's improve it and clearly state that the notebook demonstrates how to use NNCF at post-training, i.e. without the training pipeline.

Reply:
Done